### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/web` (everything except web-in-60)

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4576]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4576|
|Type|enhancement|
|Created|2010-07-11 07:08:42Z|
|Branch|https://github.com/twisted/twisted/tree/sphinx-web-4576|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for `projects/web` (everything except web-in-60)

Attachments:

* [web.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f6f/f6fb54bc99e7a6560cc4b6a1e3d4fb59333ada4c/5b75af861d5cc6001c08bd31e12a3ae31cc2e9ef.patch) (20454 bytes) - added by jdb on 2010-09-10 12:04:44Z - Adapted the lore input files so that no <code> or <em> tags are the first characters of a line. This fixes the rst and html final output.
* [web.xhtml.2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/f6f/f6fb54bc99e7a6560cc4b6a1e3d4fb59333ada4c/6fcd1245fb8be6ce3cb6b78f0a5cd58f2548eecd.patch) (20451 bytes) - added by jdb on 2010-09-10 16:30:58Z - correction in the patch

<details><summary>Searchable metadata</summary>

```
trac-id__4576 4576
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_sphinx_web_4576 branches/sphinx-web-4576
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278832122000000 1278832122000000
changetime__1284821584000000 1284821584000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jdb removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [30031]) Branching to 'sphinx-web-4576'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [30032]) Apply web.xhtml.2.patch 

Refs [#4576](https://github.com/twisted/twisted/issues/4576)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

(In [30034]) Fix a couple more simple things

refs [#4576](https://github.com/twisted/twisted/issues/4576)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks, looks good.  I fixed a couple more small things.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [30035]) Merge sphinx-web-4576

Author: jdb
Reviewer: exarkun
Fixes: [#4576](https://github.com/twisted/twisted/issues/4576)

Tweaks to the Lore markup in the Web documentation to make it
more amenable to conversion to Sphinx.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
